precompiles/assets-erc20/ERC20.sol

```diff
@@ -1,11 +1,11 @@
  pragma solidity ^0.8.0;
 
     /**
-     * @title ERC20 interface
+     * @title ERC20 interface extended by additional methods
      * @dev see https://github.com/ethereum/EIPs/issues/20
      * @dev copied from https://github.com/OpenZeppelin/openzeppelin-contracts
      */
-    interface IERC20 {
+    interface IERC20Plus {
```

Comment:
Let’s keep IERC20 as is, it’s Ethereum standard, just add one more interface with new name.

Reply:
Maybe plus is not best name. Something like `xc20mintable` sounds more explanatory